### DISCUSSION TITLE
Failed to archive error

### DISCUSSION THREAD

**Discussion by NickWoodUK** [codemagic-ci-cd/codemagic-docs#1074] [Q&A] [answered]
Hi Everyone.

I am running an iOS build and running into the following error, any help would be much appreciated! :) 

Check MyFirstReactApp.xcworkspace build settings
Execute "xcodebuild -workspace /Users/builder/clone/ios/MyFirstReactApp.xcworkspace -scheme MyFirstReactApp -showBuildSettings"

Archive MyFirstReactApp.xcworkspace
Execute "xcodebuild -workspace /Users/builder/clone/ios/MyFirstReactApp.xcworkspace -scheme MyFirstReactApp -archivePath build/ios/xcarchive/MyFirstReactApp_14954jqs.xcarchive archive COMPILER_INDEX_STORE_ENABLE=NO"

    Run script build phase 'Start Packager' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'MyFirstReactApp' from project 'MyFirstReactApp')
    Run script build phase 'Bundle React Native code and images' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'MyFirstReactApp' from project 'MyFirstReactApp')
Failed to archive /Users/builder/clone/ios/MyFirstReactApp.xcworkspace

**Comment by himesh-cm** [comment]
Hi @NickWoodUK, Can you please share the build id?

**Comment by gopherios** [comment, marked as answer]
Try to set env in codemagic.yaml -> xcode: 13.4.1

**Reply by NickWoodUK** [reply]
Thanks. That let me get to the underlying issue.
